docs/Using-Operations-Service.md

```diff
@@ -7,6 +7,8 @@
 - [Start Periodic Polling](#start-periodic-polling)
 - [Approve an Operation](#approve-an-operation)
 - [Reject an Operation](#reject-an-operation)
+- [Detail of an Operation](#detail-of-an-operation)
```

Comment:
an Operation is grammatically wrong, it should be `the` or use just `Operation Detail`